assets/src/stories-editor/components/custom-video-block-edit.js

```diff
@@ -266,6 +277,12 @@ class CustomVideoBlockEdit extends Component {
 							onChange={ this.toggleAttribute( 'loop' ) }
 							checked={ loop }
 						/>
+						<TextControl
+							label={ __( 'Accessibility label', 'amp' ) }
```

Comment:
```suggestion
							label={ __( 'Accessibility Label', 'amp' ) }
```

To match `Video Settings` and other labels here. `Assistive Text` could work too.

Reply:
Updated the wording now to align with #3214.